robotbase/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -5,14 +5,24 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.Constants.*;
+import frc.robot.commands.PIDTestCommand;
+import frc.robot.util.Utility;
 import frc.robot.commands.DefaultDriveCommand;
 
 public class RobotContainer {
   private AutoCommandChooser m_autoCommandChooser;
 
   public RobotContainer() {
     m_autoCommandChooser = new AutoCommandChooser();
+    Robot.drive.setDefaultCommand(new DefaultDriveCommand());
+  }
+
+  // private final Telemetry logger = new Telemetry(); // This puts a TON of stuff on shuffleboard.
 
+  private void configureBindings() {
     Robot.drive.setDefaultCommand(new DefaultDriveCommand());
   }
```

Comment:
Don't call Robot.drive.setDefaultCommand twice. And if we're not using this method, we can just delete it.